design-documents/graph-ql/coverage/quote.graphqls

```diff
@@ -83,7 +83,10 @@ input CartItemUpdateInput {
 
 input RemoveItemFromCartInput {
     cart_id: String!
-    cart_item_id: Int!
+    # Implementation Note: The resolver should accept either `cart_item_id` or `cart_item_uid`,
+    # but _must_ return an error when both fields are provided in a request
+    cart_item_id: Int! @deprecated(reason: "Use `RemoveItemFromCartInput.cart_item_uid` instead")
+    cart_item_uid: ID! @doc(description: "Unique Identifier from objects implementing `CartItemInterface`")
```

Comment:
Same comment as for `CartItemUpdateInput`

Reply:
👍 We'll keep discussing the general problem in that thread, and apply it across this PR once we reach consensus

Reply:
Fixed in a66f166